BUILD

```diff
@@ -6,7 +6,13 @@ load("@io_bazel_rules_go//go:def.bzl", "gazelle", "go_prefix")
 
 go_prefix("istio.io/mixer")
 
-gazelle(name = "gazelle")
+gazelle(
+    name = "gazelle",
+    args = [
+        "-build_file_name",
+        "BUILD,BUILD.bazel",
```

Comment:
Now we don't have files named `BUILD.bazel` anymore?  Also, that would be the default value for gazelle. Why should this be specified explicitly?

Reply:
Default is `BUILD.bazel` which is less popular in this repo.